single-exposiciones.php

```diff
@@ -0,0 +1,124 @@
+<?php
+get_header();
+
+
+if (qtranxf_getLanguage() == 'es') {
+  $locale = 'es_ES';
+} else {
+  $locale = 'en_US';
+}
+
+\Moment\Moment::setLocale($locale);
+
+$date_format = 'd M., y';
+
+?>
+
+<!-- main content -->
+
+<main id="main-content" class="container">
+
+  <div class="row">
+
+<?php
+if( have_posts() ) {
+  while( have_posts() ) {
+    the_post();
+    $meta = get_post_meta($post->ID);
+    $post_terms = get_the_terms($post->ID, 'tipo_de_exposicion');
+?>
+
+    <div class="col col-6">
+
+      <ul class="filter-menu">
+      <?php
+        $exhibition_types = get_terms('tipo_de_exposicion');
+
+        if ($exhibition_types) {
+          foreach ($exhibition_types as $type) {
+            if ($post_terms[0]->ID === $type->ID) {
```

Comment:
this part of the code gives an error if `$post_terms` is empty

also is not working well
@varffs did u actually test this code? i think is `term_id` not `ID`…